modules/auxiliary/dos/http/slowloris.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 # Note, works with both python 2.7 and 3
```

Comment:
Presumably worth deleting these lines? 👀

Reply:
Damn, I had deleted those originally, then reverted everything due to trying to sort out the whitespace issues and clearly forgot the second time round. :sweat_smile: